modules/wizard/view-wizard-step-languages.php

```diff
@@ -5,14 +5,15 @@
  * @package Polylang
  *
  * @since 2.7
+ *
+ * @var PLL_Admin_Model $model   `PLL_Admin_Model` instance.
+ * @var array           $options List of Polylang options.
```

Comment:
Shouldn't our new `Options` class instead or is it too soon and require many changes elsewhere?

Reply:
`$options` is now useless since using `WP_Syntex\Polylang\Model\Languages::get_default()` method.

Reply:
Removed in 60f12a0f0250bbbaa4b1c5972142a572ee14ff18